src/main/java/istjbot/command/ListCommand.java

```diff
@@ -0,0 +1,29 @@
+package istjbot.command;
+
+import istjbot.exception.BotException;
+import istjbot.storage.Storage;
+import istjbot.task.TaskList;
+import istjbot.ui.Ui;
+
+public class ListCommand extends Command {
+    public ListCommand(CommandEnum commandEnum, String fullCommand) {
+        super(commandEnum, fullCommand);
+    }
+
+    @Override
+    public boolean isExit() {
+        return false;
+    }
+
+    public void execute(TaskList tasks, Ui ui, Storage storage) throws BotException {
```

Comment:
Perhaps add a comment explaining the execute method for each Command class?
I noticed this in other Command classes as well